.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
We may not want to do this daily. What are the options?

Reply:
daily—runs on every weekday, Monday to Friday.
weekly—runs once each week. By default, this is on Monday. (can modify this)
monthly—runs once each month. This is on the first day of the month.